examples/undocumented/libshogun/mathematics_lapack.cpp

```diff
@@ -68,9 +69,9 @@ void test_matrix_multiply()
 	SGMatrix<float64_t> A(n, m);
 	SGVector<float64_t>::range_fill_vector(A.matrix, m*n);
 	SGMatrix<float64_t>::display_matrix(I, "I");
-	SGMatrix<float64_t>::transpose_matrix(A.matrix, A.num_rows, A.num_cols);
```

Comment:
this test should be done without transposes, because the multiply method accepts flags for that